src/System.ServiceModel.Tests.Common/src/BaseAddress.cs

```diff
@@ -5,6 +5,29 @@
 
 public static class BaseAddress
 {
+#if USE_FIDDLER
+    // Base address for testing nonexistent endpoints
```

Comment:
This approach is really a special case of what I intended in https://github.com/dotnet/wcf/issues/29.  So if we do this approach, we will need to back it out when we do it right -- which I think consists of propagating an MSBuild property ("WcfBaseURL"?) into the build.  It also is affected by the Bridge conversations, because the URI's will be provided by Bridge.  So it probably means we need to be able to specify the URL of the Bridge itself, as well as the base address we want Bridge to prepend if it chooses localhost.  This means we should really drive this requirement into the Bridge discussion, so that we can best choose how Bridge gets configured.